README.md

```diff
@@ -8,7 +8,6 @@
 - Add node.js Express / PostgreSQL example
 - Add React (CRA) example
 - Add React (Next.js) example
-- Add Rails example
```

Comment:
Could you add a link to the rails example to line 4 of the readme? Also could we move the rails example to `./example-rails/`?